### DISCUSSION TITLE
transition from left to right

### DISCUSSION THREAD

**Discussion by 549393092** [quasarframework/quasar#13240] [General - Components / Directives / etc] [answered]
Dear Quasar team,

So far as I know, the transition is all basically from top to bottom, like QSlideTransition or QExpansionItem.
Is there the component that supports from left to right, thanks!

**Comment by MilosPaunovic** [comment, marked as answer]
Take a look at this:
[https://quasar.dev/options/transitions](https://quasar.dev/options/transitions)

**Reply by 549393092** [reply]
@MilosPaunovic Thanks!
I tried to use the syntax but it's not working.
I did a demo, please see https://codepen.io/Stanley-549393092/pen/PoEvwyy?editors=1111
Currently I would like to show/hide card-section with transition from left to right.
I am not sure if it's possible.

**Reply by MilosPaunovic** [reply]
You can't achive that for card element, as it does not mention it as a posibility in the API section.

> There are a few Quasar components that mention about transitions through transition-show/transition-hide or transition-prev/transition-next or simply transition props. We’re going to showcase these transitions here.

But, you can easily make it using Vues transitions, something like this:
[https://codepen.io/milospaunovic/pen/XWVwXPd?editors=1111](https://codepen.io/milospaunovic/pen/XWVwXPd?editors=1111)

**Reply by 549393092** [reply]
@MilosPaunovic Thanks for your help! But I am afraid the result is not so good because the transition is out the border of card-section.
Actually I want something like Q-Drawer but it has to inside Q-Layout. As you know, there must be a Q-Layout for the main page, and for every application page, I am not sure if it's the best practice to define Q-Layout again?
